packages/atomic-react/README.md

```diff
@@ -295,90 +296,94 @@ import {AtomicSearchInterface} from '@coveo/atomic-react';
-
-You can do that by running `npm view @coveo/atomic-react`, and then identifying the `@coveo/atomic` dependency:
-
-```
-npm view @coveo/atomic-react
-
-
-@coveo/atomic-react@1.23.11 | Proprietary | deps: 1 | versions: 209
-React specific wrapper for the Atomic component library
-https://github.com/coveo/ui-kit#readme
+First, identify the required versions of `@coveo/atomic`, `@coveo/headless`, `react` and `react-dom` that are used by `@coveo-atomic-react`.
 
-dist
-.tarball: https://registry.npmjs.org/@coveo/atomic-react/-/atomic-react-1.23.11.tgz
-.shasum: 64ba0a5f686e3638b1180910f6fed5e4980bd9e2
-.integrity: sha512-/qI5O7SWNinBYWAedRW62ko0oF4c/hfapgqKe9xSq/NJfg+KBzcR3SbdyhpaUAWpPX1910GP9cMyr4wf8irJnQ==
-.unpackedSize: 771.1 kB
+You can do that by running `npm view @coveo/atomic-react@latest version dependencies devDependencies peerDependencies`, like so:
 
-dependencies:
-@coveo/atomic: 1.108.2
 ```
-
-In the above example, you can see that of the time of this writing, the current latest version of `@coveo/atomic-react` is `1.23.11`, and that it is using `@coveo/atomic` at version `1.108.2`.
-
-Then, you need to do the same to find the matching version of `@coveo/headless` used by the target`@coveo/atomic` version.
-
-Using the above example with `@coveo/atomic@1.108.2`:
-
-```bash
-npm view @coveo/atomic@1.108.2
-
-
-@coveo/atomic@1.108.2 | Apache-2.0 | deps: 15 | versions: 728
-A web-component library for building modern UIs interfacing with the Coveo platform
-https://docs.coveo.com/en/atomic/latest/
-
-dist
-.tarball: https://registry.npmjs.org/@coveo/atomic/-/atomic-1.108.2.tgz
-.shasum: 6e844073f55f10328b4b828c0190b8fe31403fbf
-.integrity: sha512-LhFR6k7NdGi8pYHmWOq3gM3rE0zeTKc6biTa1Z+ZKr5TEXIr97rpxi1HSnViQUowQYYCcY44dvsAYz2yyPvXTA==
-.unpackedSize: 87.1 MB
-
-dependencies:
-@coveo/bueno: 0.42.1                  @stencil/store: 1.5.0                 focus-visible: 5.2.0
-@coveo/headless: 1.103.3              coveo-styleguide: 9.34.4              i18next-http-backend: 1.4.1
-@popperjs/core: ^2.11.6               dayjs: 1.11.5                         i18next: 20.6.1
-@salesforce-ux/design-system: ^2.16.1 dompurify: 2.3.10                     stencil-inline-svg: 1.1.0
-@stencil/core: 2.17.3                 escape-html: 1.0.3                    ts-debounce: ^4.0.0
-
+$ npm view @coveo/atomic-react@latest version dependencies devDependencies
+
+version = '2.1.35'
+dependencies = { '@coveo/atomic': '2.19.16' }
+devDependencies = {
+  '@coveo/headless': '2.8.10',
+  '@rollup/plugin-commonjs': '^22.0.2',
+  '@rollup/plugin-node-resolve': '^14.1.0',
+  '@rollup/plugin-replace': '^4.0.0',
+  '@rollup/plugin-typescript': '^8.5.0',
+  '@types/node': '15.14.9',
+  '@types/react': '18.0.17',
+  '@types/react-dom': '18.0.6',
+  ncp: '2.0.0',
+  react: '18.2.0',
+  'react-dom': '18.2.0',
+  rollup: '^2.79.0',
+  'rollup-plugin-polyfill-node': '^0.10.2',
+  'rollup-plugin-terser': '^7.0.2'
+}
+peerDependencies = {
+  '@coveo/headless': '^2.0.0',
```

Comment:
**Caveat**: currently, our `peerDependencies` indicate that any version of Headless would work, but in reality, only `2.8.10` would work. This will be fixed with the new versioning process.